### PR TITLE
Consume source-build/branch/NuGet.config

### PR DESCRIPTION
**Pull request by RheaAyase**
@omajid thoughts?

### CODE REVIEW

**Review by omajid — APPROVED** [review]
LGTM!

File: Turkey.Tests/NuGetTest.cs [hunk 1]
```diff
@@ -58,19 +58,23 @@ public async Task LivePackagesAreIdentifiedCorrectly(string name, string version
...
+        [InlineData(new string[] { "foo" , "bar"}, "<add key=\"0\" value=\"foo\" /> <add key=\"1\" value=\"bar\" />", "5.0")]
+        public async Task NuGetConfigIsGeneratedCorrectly(string[] urls, string feedParts, string version)
         {
             using (var http = new HttpClient())
             {
                 NuGet nuget = new NuGet(http);
 
                 var expectedConfig = "<?xml version=\"1.0\" encoding=\"utf-8\"?> <configuration> <packageSources> " + feedParts + " </packageSources> </configuration>";
                 var cleanedExpectedConfig = Regex.Replace(expectedConfig, @"\s+", " ");
+                var nugetConfig = await new SourceBuild(http).GetNuGetConfigAsync(Version.Parse(version));
```
Comment (omajid) [inline Turkey.Tests/NuGetTest.cs:75]: This unit test is now dependent on the the network. Can somehow remove the dependency?

Comment (RheaAyase) [reply Turkey.Tests/NuGetTest.cs:75]: mmmmhmmmmmmm

Comment (RheaAyase) [reply Turkey.Tests/NuGetTest.cs:75]: Well I'd just remove the changes then. Faking the file makes no sense, it's already doing that.

**Comment by omajid** [comment]
Done: https://github.com/redhat-developer/dotnet-bunny/releases/tag/v8

**Pull request merged**